src/contexts/MusicPlayerStateContext.jsx

```diff
@@ -44,7 +45,7 @@ function MusicPlayerStateContextProvider({ children }) {
      * @param {Number} index The index of a song in a playlist
      */
     function chooseSong(index) {
-        setSongIndex(index)
+        setCurrentSongIndex(index)
```

Comment:
(question): What is the difference between the songIndex state and the currentSongIndex state?